### ISSUE TITLE
Looks like typo. The potential bug

### ISSUE THREAD

**Issue by m-chichikalov** [STM32-SDR/STM32-SDR#34]
Hi guys. Take a look -->
https://github.com/STM32-SDR/STM32-SDR/blob/75013f7bc05bd95e8a19f533c182579c29c17052/STM32-SDR/code/main/src/Text_Enter.c#L913

Probably one curly bracket is missed...
73!

**Comment by DrGeeks** [comment]
Good catch and shows someone is paying attention! Correct that there are (2) brackets missing. It appears that it would allow the cursor to move beyond the limits when editing a tag field. I'm not sure if anyone is using the macro feature, but even if they do, it wouldn't be noticed unless they try to scroll left/right beyond the end of the text. 

I'll correct it in the next version. It may be some time before I get around to another update. Feel free to make the correction yourself.
